tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "include": [
+    "./src/**/*",
+    "index.d.ts"
```

Comment:
Basically copied this `tsconfig.json` from [the Gatsby with Typescript example Github](https://github.com/gatsbyjs/gatsby/blob/master/examples/using-typescript/tsconfig.json) - this file is needed to use the added `index.d.ts` file.

Reply:
weird extension I've never seen that

Reply:
Ya, apparently it's a TypeScript declaration files - [there's some docs](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html) if you're interested in reading into it. I haven't really read into myself but could be useful to know